ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableFlatEvents.scala

```diff
@@ -138,8 +140,7 @@ private[events] object EventsTableFlatEvents {
           join parameters on
               (participant_pruned_up_to_inclusive is null or event_offset > participant_pruned_up_to_inclusive)
               and event_offset <= ledger_end
-          where transaction_id = $transactionId and #$witnessesWhereClause
-          group by (#$groupByColumns)
+          where transaction_id = $transactionId and #$witnessesWhereClause and event_kind != 0 -- we do not want to fetch divulgence events
```

Comment:
Do we have a test that fails if we don't have this restriction? If not, let's add one when implementing this properly.

Reply:
added a todo